packages/shorebird_cli/lib/src/commands/patch_new/android_patcher.dart

```diff
@@ -166,4 +170,25 @@ Looked in:
       artifact.path,
     );
   }
+
+  @override
+  Future<CreatePatchMetadata> createPatchMetadata(DiffStatus diffStatus) async {
+    final allowAssetDiffs = argResults['allow-asset-diffs'] == true;
+    final allowNativeDiffs = argResults['allow-native-diffs'] == true;
```

Comment:
These would probably be better as a getter on `Patcher`

Reply:
Sounds good, will update 👍